test/endpoints/server-side.js

```diff
@@ -215,6 +225,9 @@ describe('endpoints#serverSide', function() {
   });
 
   it('should throw on server side queue timeout error', function(done) {
+
+    console.log(' ');
```

Comment:
It will fail with a timeout: 

```
1) endpoints#serverSide should throw on server side queue timeout error:
     Error: timeout of 2000ms exceeded
      at null.<anonymous> (/Users/nvcexploder/src/git/hula-hoop/node_modules/mocha/lib/runnable.js:158:19)
      at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)
```